.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - checkout
       - restore_cache: &restore_cache
           keys:
-            - v4-{{ .Branch }}-{{ checksum "setup.py" }}
+            - v5-{{ .Branch }}-{{ checksum "setup.py" }}
```

Comment:
there was some build issue that bursting the cache seemed to solve, hence this change